ci-operator/config/openshift/jenkins/master.json

```diff
@@ -0,0 +1,172 @@
+{
+    "tag_specification": {
+        "cluster": "https://api.ci.openshift.org",
+        "namespace": "openshift",
+        "name": "jenkins",
+        "tag": "",
+        "tag_overrides": {}
+    },
+    "base_images": {
+        "base": {
+            "cluster": "https://api.ci.openshift.org",
+            "namespace": "openshift",
+            "name": "origin-v3.11",
```

Comment:
i'm probably totally out of the loop on how this works but i'd hope that this would not be hardcoded in jobs, seems like it should be a variable that gets set based on the release branch being tested.

Reply:
I config file per release branch is the current pattern.

See the contents of https://github.com/openshift/release/tree/master/ci-operator/config/openshift/image-registry as a comparative example.